distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -995,6 +951,82 @@ requiredAttributes=["/\=","/admin\={http://schemas.xmlsoap.org/ws/2005/05/identi
 whiteListContexts=["/services/SecurityTokenService","/services/internal/metrics","/services/saml","/proxy","/services/csw"]
 ----
 
+==== Configuring Global System Properties
+The `system.properties` file found under `$DDF_HOME/etc` contains properties that will be made available through the system properties on startup. After changing any of these properties you will need to restart the system. If you change the 'hostname' property you will also need to configure the certs as described in Configuring {branding} with New Certificates.
+The system by default uses both http and https so both httpsPort and httpPort need to be specified. The protocol and port properties are the defaults the system should use in places where either http or https could be valid.
+----
+org.codice.ddf.system.protocol=https://        #one of http:// or https://
+org.codice.ddf.system.hostname=localhost       #should be the fully qualified domain name
+org.codice.ddf.system.httpsPort=8993           #secure port
+org.codice.ddf.system.httpPort=8181            #public port
+org.codice.ddf.system.port=8993                #default port corresponding to the protocol selected. Should match the httpPort or httpsPort
+org.codice.ddf.system.rootContext=/services    #the root context that services will be made available under
+
+org.codice.ddf.system.siteName=${sitename.default}        #the name of this instance
+org.codice.ddf.system.siteContact=                        #contact for this instance
+org.codice.ddf.system.version=${project.version}          #this instances version
+org.codice.ddf.system.organization=${organization.name}   #who owns/runs this instance
+----
+
+The above properties (along with any other system properties) are available to be used as variable parameters in input url fields within the Admin Console. For example if you wanted to enter the url for the local csw service you could write
+----
+${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:${org.codice.ddf.system.port}${org.codice.ddf.system.rootContext}/csw
+----
+
+instead of
+
+-----
+https://localhost:8993/services/csw
+-----
+
+The variable version is longer but will not need to be changed if the system host, port or root context changes.
+
+
+==== HTTP Port Configuration
+[IMPORTANT]
+====
+Do not use the Admin Console to change the HTTP port. While the Admin Console's Pax Web Runtime offers this configuration option, it has proven to be unreliable and may crash the system.
+====
+
+====== Multiple Local {branding} Nodes
+Edit the port numbers in the files in the {branding} install folder. The line numbers relate to 2.1.X releases.
```

Comment:
We don't have the line numbers anymore. Remove "The line numbers relate to 2.1.X releases.".